src/edge/importer.py

```diff
@@ -107,9 +110,30 @@ def build_fragment(self):
         prev = None
         flen = 0
         seqlen = len(self.__sequence)
+
+	# divide chunks bigger than a certain threshold to smaller chunks, to
+	# allow insertion of sequence into database. e.g. MySQL has a packet
+	# size that prevents chunks that are too large from being inserted.
+        cs_limit = 1000000
+        new_chunk_sizes = []
+        for original_chunk_size in chunk_sizes:
+            if original_chunk_size < cs_limit:
+                new_chunk_sizes.append(original_chunk_size)
+            else:
+                divided_chunks = []
+                while original_chunk_size > 0:
+                    divided_chunks.append(min(original_chunk_size, cs_limit))
+                    original_chunk_size -= cs_limit
+                new_chunk_sizes.extend(divided_chunks)
+        chunk_sizes = new_chunk_sizes
+
         for sz in chunk_sizes:
```

Comment:
Do you mind updating this sz to be something like chunk_size while we are here? Thanks!